components/identity-mgt-new/org.wso2.carbon.identity.mgt.admin.ui/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Check for correct copyright header in each file, including POMs and resource files.